changelog/@unreleased/pr-5489.v2.yml

```diff
@@ -0,0 +1,22 @@
+type: improvement
+improvement:
+  description: |-
+    Improve utility of SQLException logging
```

Comment:
changelog needs to be shortened - usually it is a one-liner explaining what the change does.

Reply:
Done